docs/reference/plugins.rst

```diff
@@ -301,6 +301,44 @@ for `djangocms-text-ckeditor <https://github.com/divio/djangocms-text-ckeditor/>
 
 See also: `text_enabled`_
 
+in_text_editor_repr
+-------------------
+
+When a plugin can be inserted as a child in a text plugin, this method provides
+the representation the text plugin must use.  For example, if
+`in_text_editor_tag`_ returns ``'span'``, parent plugins will encapsulate the
+plugin this way::
+
+    <span data-djangocms-plugin-id="the_plugin_id">whatever string returned by in_text_editor_repr</span>
+
+:meth:`in_text_editor_repr` takes 2 arguments:
+
+* ``instance``: The instance of the plugin model
+* ``editor_name``: The name of the parent plugin
+
+This method must return a string or ``None``. When it returns ``None``, parent
+plugins will fallback to the old way (`icon_alt`_ and `icon_src`_) which is
+deprecated since django cms 3.1 and will be removed in version 3.?.
+
+Default implementation returns ``None``.
+
+See also: `text_enabled`_, `in_text_editor_tag`_
+
+in_text_editor_tag
+------------------
+
+When a plugin can be inserted as a child in a text plugin, this method provides
+a way to choose the tag used to encapsulate the plugin representation.
+
+:meth:`in_text_editor_tag` takes 1 arguments:
+
+* ``instance``: The instance of the plugin model
+* ``editor_name``: The name of the parent plugin
+
+This method must return a string. Default implementation returns ``'div'``.
```

Comment:
In the documentation when we are refering to the python boolean true, we use ``True``. When refering to a to python string, we might want to use ``'whatever string'`` in order to show the value is a string. That said, I'm fine with ``whatever string``.